### PR TITLE
add: task description for depth estimation.

### PR DESCRIPTION
**Pull request by sayakpaul**
Closes: https://github.com/huggingface/hub-docs/issues/473

Second leg of https://github.com/huggingface/hub-docs/pull/488

### CODE REVIEW

File: tasks/src/depth-estimation/about.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Use Cases
+Depth estimation models can be used to approximate the depth of different objects present in an image.
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:2]:
```suggestion
Depth estimation models can be used to estimate the depth of different objects present in an image.
```
Sounded like an easier read 😅

File: tasks/src/depth-estimation/about.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Estimation of Volumetric Information
+Depth estimation models are widely used to study volumetric formation of objects present inside an image. This is an
+important use case in the domain of computer graphics.
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:6]:
```suggestion
```

File: tasks/src/depth-estimation/about.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+## Use Cases
+Depth estimation models can be used to approximate the depth of different objects present in an image.
+
+### Estimation of Volumetric Information
+Depth estimation models are widely used to study volumetric formation of objects present inside an image. This is an
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:5]:
```suggestion
Depth estimation models are widely used to study volumetric formation of objects present inside an image. This is an important use case in the domain of computer graphics. 
```

File: tasks/src/depth-estimation/about.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+important use case in the domain of computer graphics. 
+
+### 3D Representation
+
+Models trained for depth estimation can also be used to develop a 3D understanding from a 2D image which is another
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:10]:
```suggestion
Depth estimation models can also be used to develop a 3D representation from a 2D image.
```
Maybe you can mention how 3D representation can be used?

File: tasks/src/depth-estimation/about.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+important use case for computer graphics.
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:11]:
```suggestion
```

File: tasks/src/depth-estimation/about.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+With the `transformers` library, you can use the `depth-estimation` pipeline to infer with image classification models. You can initialize the pipeline with a model id from the Hub. If you do not provide a model id it will initialize with [Intel/dpt-large](https://huggingface.co/Intel/dpt-large) by default. When calling the pipeline you just need to specify a path, http link or an image loaded in PIL. 
+
+```python
+from transformers import pipeline
+
+estimator = pipeline("depth-estimation")
+result = estimator("http://images.cocodataset.org/val2017/000000039769.jpg")
+result
+
+# {'predicted_depth': tensor([[[ 6.3199,  6.3629,  6.4148,  ..., 10.4104, 10.5109, 10.3847],
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:24]: Would be nice to add a line below to visualize the depth image like you did in schema.

File: tasks/src/depth-estimation/about.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+#           [ 6.3850,  6.3615,  6.4166,  ..., 10.4540, 10.4384, 10.4554],
+#           [ 6.3519,  6.3176,  6.3575,  ..., 10.4247, 10.4618, 10.4257],
+#           ...,
+#           [22.3772, 22.4624, 22.4227,  ..., 22.5207, 22.5593, 22.5293],
+#           [22.5073, 22.5148, 22.5114,  ..., 22.6604, 22.6344, 22.5871],
+#           [22.5176, 22.5275, 22.5218,  ..., 22.6282, 22.6216, 22.6108]]]),
+#  'depth': <PIL.Image.Image image mode=L size=640x480 at 0x7F1A8BFE5D90>}
+```
+
+## Useful Resources
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:34]: Can you add call for contribution for this? You can see how we do this with other pages (Especially new ones)

File: tasks/src/depth-estimation/about.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+
+### Creating your own image classifier in just a few minutes
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/about.md:37]:
```suggestion
```

File: tasks/src/depth-estimation/data.ts [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+			id:          "Intel/dpt-large",
+		},
+		{
+			// TO DO: write description
+			description: "Strong Depth Estimation model trained on the KITTI dataset.",
+			id:          "vinvino02/glpn-kitti",
+		},
+	],
+	summary:      "Depth estimation is the task of predicting depth of the objects present in an image.",
+	widgetModels: ["Intel/dpt-large"],
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/data.ts:33]: As of now we don't have a widget for this model.
```suggestion
	widgetModels: [""],
```

File: tasks/src/depth-estimation/data.ts [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+			},
+		],
+		outputs: [
+			{
+				filename: "depth-estimation-output.png",
+				type:     "img",
+			},
+		],
+	},
+	metrics: [],
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/data.ts:19]: I think you can write something like Contribute a metric so people will be contributing it. (same applies for dataset, you can link this dataset if there's none huggingface-projects/contribute-a-dataset)

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for working on this!

File: tasks/src/depth-estimation/data.ts (hunk 10 above)
Comment (sayakpaul) [reply tasks/src/depth-estimation/data.ts:19]: Are there any examples I can refer to?

File: tasks/src/depth-estimation/data.ts (hunk 9 above)
Comment (sayakpaul) [reply tasks/src/depth-estimation/data.ts:33]: Why?

File: tasks/src/depth-estimation/about.md (hunk 7 above)
Comment (sayakpaul) [reply tasks/src/depth-estimation/about.md:32]: Added a comment.

File: tasks/src/depth-estimation/about.md (hunk 6 above)
Comment (sayakpaul) [reply tasks/src/depth-estimation/about.md:22]: Added a comment.

File: tasks/src/depth-estimation/about.md (hunk 7 above)
Comment (sayakpaul) [reply tasks/src/depth-estimation/about.md:32]: Added a comment.

File: tasks/src/depth-estimation/data.ts (hunk 9 above)
Comment (merveenoyan) [reply tasks/src/depth-estimation/data.ts:33]: We don't have the widget for the model.
Ekran Resmi 2022-11-21 18 01 55

**Review by merveenoyan — APPROVED** [review]
LGTM, please address one comment I left and we'll be good to go.

**Comment by sayakpaul** [comment]
@merveenoyan thanks! See if it's good to go now.

**Comment by sayakpaul** [comment]
Merging in 3...2...1...

**Pull request merged**